### DISCUSSION TITLE
ONLY M vs. VLC Media Player.

### DISCUSSION THREAD

**Discussion by joelgray76** [AntonyCorbett/OnlyM#468] [Q&A]
Hey brothers,

Can you share your experiences with the benefits of ONLY M vs. VLC media player?

The guidelines suggest VLC but in most congregations we used ONLY M without a problem.

What are the pros and cons of each?

**Comment by scottys94** [comment]
@joelgray76 We use OnlyM because of its simple interface and ease of use.
